test/tools/unit_test_runner.d

```diff
@@ -215,6 +215,18 @@ void writeRunnerFile(Range)(Range moduleNames, string path, string filter)
 
             return result;
         }
+
+        version (D_Coverage)
+        shared static this()
+        {
+            import core.runtime;
+
+            static immutable sourcePath = `%s`;
```

Comment:
Small update: The path is now set by `unit_test_runner` s.t. it also works when `RESULTS_DIR` is located outside of `test`.

Reply:
Ah, got it, never mind. Probably use `"` and escape `\"`.

Reply:
`%s` is replaced with `projectRootDir`. The string literal explicitly uses ``` s.t. it doesn't need to deal with the other escape sequences (e,g, `\` on windows)

Reply:
FYI: This also supports file paths with spaces, even though the `unit` tests cannot be built in that case due to this [cmdfile bug](https://issues.dlang.org/show_bug.cgi?id=22773)